modules/apps/transfer/ibc_module.go

```diff
@@ -181,7 +180,7 @@ func (im IBCModule) OnRecvPacket(
 	var data types.FungibleTokenPacketData
 	var ackErr error
 	if err := types.ModuleCdc.UnmarshalJSON(packet.GetData(), &data); err != nil {
-		ackErr = sdkerrors.Wrapf(icatypes.ErrInvalidChannelFlow, "cannot unmarshal ICS-20 transfer packet data")
+		ackErr = sdkerrors.Wrapf(types.ErrInvalidChannelFlow, "cannot unmarshal ICS-20 transfer packet data")
```

Comment:
I know we added this in the other PR also but I don't really think this error message fits here. The invalid channel flow for ICA was implying that for example, you can't do controller handshake flow on the host module and vice versa.